Sources/SQLKit/Builders/SQLSelectBuilder.swift

```diff
@@ -17,6 +17,24 @@ public final class SQLSelectBuilder: SQLQueryFetcher, SQLQueryBuilder, SQLPredic
         self.database = database
     }
     
+    public func distinct() -> Self {
+        self.select.isDistinct = true
+        return self
+    }
+    
+    public func distinct(on columns: String...) -> Self {
+        self.select.isDistinct = true
+        self.select.columns = []
+        columns.forEach { _ = self.column($0) }
```

Comment:
In keeping with the style used elsewhere in SQLKit, I suggest replacing these two lines with `self.select.columns = columns.map { SQLColumn($0) }`

Reply:
The reason I didn't do this is because I wanted the `"*"` check to still work without having to copy a lot of code :)

Reply:
Also keep the String -> SQLColumn logic in one place

Reply:
Fair point, objection withdrawn! :)